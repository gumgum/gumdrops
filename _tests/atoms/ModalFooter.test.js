import renderer from 'react-test-renderer';
import React from 'react';
import ModalFooter from '../../components/atoms/ModalFooter';

const defaultProps = {
    className: 'modal-class',
    style: { width: '100px' }
};

test('Expect <ModalFooter> to render properly', () => {
    const tree = renderer
        .create(
            <ModalFooter {...defaultProps}>
                <div>My modal content</div>
            </ModalFooter>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
