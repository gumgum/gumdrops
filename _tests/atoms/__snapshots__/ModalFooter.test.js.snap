// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <ModalFooter> to render properly 1`] = `
<div
  className="gds-modal__footer -text-right modal-class"
  style={
    {
      "width": "100px",
    }
  }
>
  <div>
    My modal content
  </div>
</div>
`;
